Source/WebCore/platform/graphics/ca/PlatformCALayer.mm

```diff
@@ -149,7 +149,7 @@
     };
 
     auto attributes = adoptCF(CFDictionaryCreate(kCFAllocatorDefault, keys, values, std::size(keys), &kCFTypeDictionaryKeyCallBacks, &kCFTypeDictionaryValueCallBacks));
-    auto string = adoptCF(CFStringCreateWithBytesNoCopy(kCFAllocatorDefault, reinterpret_cast<const UInt8*>(text), length, kCFStringEncodingUTF8, false, kCFAllocatorNull));
+    auto string = adoptCF(CFStringCreateWithBytesNoCopy(kCFAllocatorDefault, text.data(), text.size(), kCFStringEncodingUTF8, false, kCFAllocatorNull));
```

Comment:
Since this treats the text as UTF-8 the character type should not be `LChar`. Maybe it's ASCII only? Could use `char8_t`.

Reply:
There is a single call site and it looks like the text is passing an integer converted to a String:
```
        auto repaintCount = m_tileRepaintCounts.count(tileLayer);
        std::array<LChar, lengthOfIntegerAsString(std::numeric_limits<decltype(repaintCount)>::max())> repaintCountCharacters;
        writeIntegerToBuffer(repaintCount, repaintCountCharacters.data());
```

Reply:
That said, in theory, nothing prevents someone from calling this function with a UTF8 string.

Reply:
I updated the PR to use char8_t.